packages/components/pie-thumbnail/src/index.ts

```diff
@@ -13,8 +17,33 @@ const componentSelector = 'pie-thumbnail';
  * @tagname pie-thumbnail
  */
 export class PieThumbnail extends LitElement implements ThumbnailProps {
+    @property({ type: String })
+    @validPropertyValues(componentSelector, variants, defaultProps.variant)
+    public variant = defaultProps.variant;
+
+    @property({ type: String })
+    public src = defaultProps.src;
+
+    @property({ type: String })
+    public alt = defaultProps.alt;
+
     render () {
-        return html`<h1 data-test-id="pie-thumbnail">Hello world!</h1>`;
+        const {
+            variant,
+            src,
+            alt,
+        } = this;
+
+        const wrapperClasses = {
+            'c-thumbnail': true,
+            [`c-thumbnail--${variant}`]: true,
+        };
+
+        return html`
+            <div data-test-id="pie-thumbnail" class="${classMap(wrapperClasses)}">
+                <img data-test-id="pie-thumbnail-img" src="${src}" class="c-thumbnail-img" alt="${alt}">
```

Comment:
@raoufswe I am kind of wondering if it would make more sense to provide a slot for an `img` tag so we aren't just passing those attributes down as props. [Soectrum](https://opensource.adobe.com/spectrum-web-components/components/thumbnail/) appears to do this

Reply:
that's a good idea but our setup and use cases are a bit different and I believe it will be better if we control the img tag within the component such as handling placeholder props, defaulting to the placeholder if an error arises, and dispatching the error and changing the aspect ratio of the image.